src/rust/dbv2/migrations/2023-11-16-103707_market_volume/up.sql

```diff
@@ -0,0 +1,115 @@
+-- Your SQL goes here
+CREATE MATERIALIZED VIEW aggregator.markets_24h_data AS
+    WITH fills AS(
+        SELECT
+            *
+        FROM
+            fill_events
+        WHERE
+            "time" >= CURRENT_TIMESTAMP - '1 day'::interval
+            AND emit_address = maker_address
+    )
+    SELECT
+        market_id,
+        MIN(fills.price) AS min_price_24h,
+        MAX(fills.price) AS max_price_24h,
+        SUM(fills.size) AS base_volume_24h,
+        SUM(fills.size * fills.price) AS quote_volume_24h
+    FROM
+        fills
+    GROUP BY
+        market_id;
+
+
+DROP VIEW api.markets;
+
+
+CREATE VIEW api.markets AS
+    WITH max_time AS (
+        SELECT
+            MAX("time") AS mtime
+        FROM
+            fill_events
+    ),
```

Comment:
If multiple events happen in the same txn, this will pick the earliest one, correct? And thus not necessarily the latest one in terms of emission sequence?